endpoint/spec.md

```diff
@@ -36,7 +36,7 @@ this form:
   "documentation": "URL", ?
   "labels": {
     "modelversion": "1.0", ?
-    "compatiblewith": "https://github.com/xregistry/spec/blob/main/endpoint/spec.md", ?
+    "compatiblewith": "https://xregistry.soaphub.org/model/groups/endpoints", ? # TODO: Doesn't work, URL needs to change
```

Comment:
would pointing to: https://github.com/xregistry/spec/blob/main/endpoint/model.json work?

Reply:
I'll use the raw link, so it's a JSON download, not HTML